Update cloud faq article (#6528)
umbraco/UmbracoDocs: umbraco-cloud/frequently-asked-questions.md

### Can I run my high-traffic site on Umbraco Cloud?

> Currently, we have benchmarked a "well-built" site with approximately 50,000 unique visitors per day (\~1.5 million per month) that performs well. For business-critical, high-traffic sites, we recommend that you look into Umbraco Cloud Professional and Umbraco Cloud Enterprise possibly in combination with a dedicated server.

Author: ```suggestion
Currently, we have benchmarked a "well-built" site with approximately 50,000 unique visitors per day (\~1.5 million per month) that performs well. For business-critical, high-traffic sites, we recommend that you look into Umbraco Cloud Professional and Enterprise possibly in combination with a dedicated server.
```

### Can I move my site from Umbraco Cloud?

> Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. So if you decide that Umbraco Cloud isn’t right for you or your sites then you can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.

Author: ```suggestion
Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
```

> Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.

Author: ```suggestion
Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go. But we understand there are some requirements we might not be able to fulfill. So we support and encourage you to choose the best solution for your specific site needs.
```

Yes, you can. Please note that Umbraco Cloud also uses Cloudflare for DNS, so you need to enroll your hostname as 'DNS Only' with a CNAME pointing to `dns.umbraco.io`. Once you can see the hostname is marked with 'Protected' under the Project / Hostname subpage you can turn on 'Proxying' for the hostname in your Cloudflare account if you need to use specific Cloudflare features like Page Rules.

> Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account and let Umbraco Cloud handle the automatic TLS (HTTPS) certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.

Author: ```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic TLS (Transport Layer Security) /HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

> Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.

Reviewer: ```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. This lets Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

> Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic TLS (Transport Layer Security) /HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.

Author: ```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

Note, the HTTP requests headers are available on all custom hostnames created through Umbraco Cloud. But not the default hostname for the Umbraco Cloud project such as project.euwest01.umbraco.io.

### Which Umbraco versions are available on Umbraco Cloud?

> All the latest supported versions of Umbraco CMS are available on Umbraco Cloud. See the [Long-term Support & End-of-Life](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) page for more information.

Reviewer: ```suggestion
All supported versions of Umbraco CMS are available on Umbraco Cloud. See the [Long-term Support & End-of-Life](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) page for more information.
```

### My project didn't receive the auto-upgrade. Why?

> When we roll out auto-upgrades to Umbraco Cloud projects the first thing that happens is a check of all environments on a project. This check will verify whether the environments are responding and don't return an HTTP status error. If the auto-upgrader encounters HTTP status errors on any of the environments during this check, the upgrade process is aborted. Then your project will not receive the upgrade.

Reviewer: ```suggestion
When we roll out auto-upgrades to Umbraco Cloud projects the first thing that happens is a check of all environments on a project. This check will verify whether the environments are responding and do not return an HTTP status error. If the auto-upgrader encounters HTTP status errors on any of the environments during this check, the upgrade process is aborted. Your project will not receive the upgrade should this happen.
```

### Does Umbraco Cloud support TLS / HTTPS?

> Yes, in fact, Umbraco Cloud provides automatic TLS (HTTPS) certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.

Author: ```suggestion
Yes, in fact, Umbraco Cloud provides automatic TLS (Transport Layer Security) /HTTPS certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

> Yes, in fact, Umbraco Cloud provides automatic TLS (Transport Layer Security) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.

Author: ```suggestion
Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

> Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.

Author: ```suggestion
Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS)/HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

Yes, you can choose between West Europe, East US, South UK, and East Australia regions.

### Can I move my existing project created on Cloud in the EU region to other regions?

> Yes, you can move a project that was created on Umbraco Cloud in the EU region to another region by following the [migrate between regions guide](getting-started/migrate-between-regions.md).

Author: ```suggestion
Yes, you can move a project that was created on Umbraco Cloud in the EU region to another region. Navigate to the [migrate between regions guide](getting-started/migrate-between-regions.md) for more information.
```
